index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
```

Comment:
- [ ] [REQUIRED] Your `hx` tag should have any color on `style.css` file but you did not give style your `hx` tag and did not create `style.css` file. Please fix this issues!

![image](https://user-images.githubusercontent.com/63915024/185155849-04b46c26-0646-48e8-914f-5fe0979c8488.png)